packages/connectivity/connectivity_macos/lib/main.dart

```diff
@@ -0,0 +1,2 @@
+// Analyze will fail if there is no main.dart file. This file should
+// be removed once an example app has been added to connectivity_macos.
```

Comment:
You may link to this issue in the meantime. Also, you can just create any .dart file pretty much anywhere in the package, no need to make this `/lib/main.dart`; it can also be `/dummy.dart`

https://github.com/flutter/flutter/issues/51007

Reply:
`* The name of "lib/main.dart", "main", should match the name of the package, "connectivity_macos".
  This helps users know what library to import.`

🤦‍♂️